packages/paste-website/src/pages/tokens/how-to-compose-custom-ui-with-tokens.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: How to compose custom UI with tokens
+description: You may encounter times where you are required to build an experience that isn't covered by out of the box Paste components. This guide aims to help you create custom experiences whilst still using the design system.
+---
+
+import {graphql} from 'gatsby';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+
+<content>
+
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+<p>{props.pageContext.frontmatter.description}</p>
+
+</content>
+
+***
+
+<content>
+
+Paste will never aim to cover every experience that Twilio ships to customers, it will make our very small but mighty team an incredible bottle neck to you doing your job and we don't want that. We also fully expect that there will be certain, very product specific experiences that would make little sense as generic components or design patterns that can be used in any product vertical. With that in mind we have hopefully devised some ways that will help you and your team ship your products whilst still being able to use assets from the design system.
+
+## The happy path
+
+Obviously the first option is to work with your design partner and identify all the existing Paste components that can be used to create the experience. If you are struggling or have quesitons come to Design System Office Hours or post in the #help-design-system slack channel, we'll get you back on track.
```

Comment:
`quesitons` -> `questions`

Reply:
Should we link to info on Office Hours?

Reply:
> Should we link to info on Office Hours?

Do we have a link?

Reply:
https://wiki.hq.twilio.com/display/PD/Paste%3A+A+Design+System+for+Twilio#Paste:ADesignSystemforTwilio-WeeklyOfficeHours